src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+        * Build charts, dashboards, and alerts from anywhere in our platform using the NRQL console. 
+        * Take advantage of our free tutorials and New Relic University courses to learn everything you need to know about the language. 
+        * Explore your data in ways no other platform can provide and get every detail on any aspect of observability.
+        * Use NRQL to make queries and perform specific configurations via NerdGraph, our GraphQL-format API.
+    Learn more about [how NRQL works](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/), or see our [introduction to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) documentation.       
+
+    </Collapser>
+    <Collapser
+        id="Vuln-management"
+        title="Vulnerability management"
+    >
+    As your stack grows, security issues become an ever growing problem to tackle. Our vulnerability management solution gives you insight into what you need so that you can make sure to maintain compliance across your entire system.
+        * Sync up security with all of your teams from one centralized location and streamline issue management.
+        * Prevent accidental vulnerabilities and breaches across your entire organization with automatic identification. 
+        * Detect your vulnerabilities in minutes, letting you fix them before they grow into large-scale problems.
+        * See recommended updates automatically and ensure you're up-to-date on all your security needs.
+    Learn more about vulnerability management in our [short overview](https://newrelic.com/platform/vulnerability-management), or see our [introduction to vulnerability management](/docs/vulnerability-management/overview/) documentation.     
+    </Collapser>
+    <Collapser
+        id="Codestream"
+        title="CodeStream"
+    >
+    Collaboration across different development teams can be challenging, requiring developers to go back and forth between multiple tools to get anything done. New Relic's CodeStream makes it easier by allowing you to see your data, then discuss and review code, all within your IDE.
+        * See your telemetry in context, allowing developers across teams to collaborate without missing out on valuable information.
+        * Integrate with VS Code, Visual Studio, and all JetBrains editors as easily as adding an extension.
+        * View code-level metrics to fully understand production environment performance.
+        * Collaborate more easily across all your tools, such as GitHub, Slack, Teams, and email.
+    Learn more about CodeStream in our [short overview](https://newrelic.com/codestream), or see our [introduction to CodeStream](/docs/codestream/start-here/what-is-codestream/) documentation.         
```

Comment:
Same as browser, gets combined with previosu bullet.

Reply:
Should be fixed